scripts/renamer/README.md

```diff
@@ -0,0 +1,49 @@
+# Renamer
+
+This module is used to apply renaming rules to service IDs and document types.
+
+## Usage
+
+You can use it in your others scripts like this:
+
+```
+await renamer.loadRules();
+const { serviceId: renamedServiceId, documentType: renamedDocumentType } = renamer.applyRules(serviceId, documentType);
+```
+
+## Adding renaming rules
+
+### Service
+
+To rename a service, add a rule in `./rules/services.json`, for example, to rename "GoogleAds" to "Google Ads", add the following line in the file:
```

Comment:
If we call the document types renaming file `documentTypes`, why call the service name renaming file `services` and not `serviceNames`? 🙂